pkg/geo/geogfn/dwithin.go

```diff
@@ -21,14 +21,14 @@ import (
 // equivalent to Distance(a, b) <= d. Otherwise, DWithin is instead equivalent
 // to Distance(a, b) < d.
 func DWithin(
-	a *geo.Geography,
-	b *geo.Geography,
+	a geo.Geography,
+	b geo.Geography,
 	distance float64,
 	useSphereOrSpheroid UseSphereOrSpheroid,
 	exclusivity geo.FnExclusivity,
 ) (bool, error) {
 	if a.SRID() != b.SRID() {
-		return false, geo.NewMismatchingSRIDsError(a, b)
+		return false, geo.NewMismatchingSRIDsError(&a, &b)
```

Comment:
Drive-by comment: this error is probably making the local vars escape back to the heap wherever it’s used. We might want to make the args values too.

Reply:
Might address that in a follow up PR. Gotta remove the interface and use spatial object instead.

Reply:
SGTM. I did just confirm that this is the case though, so you'll want to make that change before running `sqlcmp`. As is, this might actually be a pessimization.